packages/frontapp/src/app/globalUI/Table/index.tsx

```diff
@@ -70,7 +104,7 @@ const Table: FC<{ columnTitles: string[]; rows?: ReactElement[] }> = ({
   return (
     <TableContainer>
       <List>
-        <Title>
+        <Title visible={false}>
```

Comment:
I think `visible` prop is not needed. You can make check if `Title` should be displayed, here.
Use https://www.npmjs.com/package/react-responsive to check screen-size